### ISSUE TITLE
Post process to acquire the affinity matrix

### ISSUE THREAD

**Issue by mengxianghan123** [mohsenkheirandishfard/MLRDSC#1]
Thanks for your significant work!!!
But here I got one confusing point. It seems that you calculate the affinity matrix from C as the same way in DSCNet. But in the paper, it seems to be calculated in a different way.
Could you please explain that in more details?

Thanks a lot!!!

**Comment by mohsenkheirandishfard** [comment]
Thank you very much for your interest. 
Actually, I am not sure if I understand your question correctly. As you mentioned, we are using the same functions as the ones used in DSCNet. However, the input to these functions is computed via the following formula (line 157 in EYaleB-38Subjects.py)

Coef = thrC(C.detach().cpu().numpy() + (1. / 3) * (D1 + D2 + D3).detach().cpu().numpy(), alpha)
which is aligned with Eq. 6 in the paper.

**Comment by mengxianghan123** [comment]
Thanks for your reply! I come to understand your point now!
